package.json

```diff
@@ -5,22 +5,15 @@
   "main": "index.js",
   "scripts": {
     "build": "npm run build:spec",
-    "build:spec": "node scripts/build.mjs",
-    "lint": "npm run build:spec && node scripts/validate.mjs && node scripts/graphql-validate.mjs",
+    "build:spec": "node scripts/build.js",
+    "build:docs": "npm run generate-clients && npm run build:docs:gatsby",
+    "build:docs:gatsby": "cp docs/config/gatsby-config.js build/docs/gatsby && cd build/docs/gatsby && npm install && gatsby build --prefix-paths",
```

Comment:
I think this will be cleaned up viai the generator channges that will allow you to spec paths in the config.json